src/core/Form/MultiSelect/MultiSelect/MultiSelect.md

```diff
@@ -89,15 +89,18 @@ const defaultSelectedTools = [
 
 <>
   <MultiSelect
-    labelText="MultiSelect"
+    labelText="Tools"
+    hintText="You can filter results by typing in the field"
     items={tools}
     chipListVisible={true}
     ariaChipActionLabel="Remove"
     removeAllButtonLabel="Remove all selections"
-    visualPlaceholder="Choose your tool(s)"
+    visualPlaceholder="Choose your tools"
```

Comment:
Removed the parens from this and another example as those sounded weird when read by screenreader.